include/deal.II/opencascade/utilities.h

```diff
@@ -219,6 +222,28 @@ namespace OpenCASCADE
   void create_triangulation(const TopoDS_Face &face,
                             Triangulation<2,3> &tria);
 
+
+  /**
+   * Given a Triangulation and an optional Mapping, create a vector of smooth
+   * curves that interpolate the connected parts of the boundary vertices of
+   * the Triangulation and return them as a vector of TopoDS_Edge.
+   *
+   * The returned curves are ordered with respect to the indices of the faces
+   * that make up the triangulation boundary, i.e., the first curve is the one
+   * extracted starting from the face with the lowest index, and so on.
+   *
+   * @param[input] triangulation Input triangulation
+   * @param[input] mapping Optional input mapping
+   * @return An std::vector of TopoDS_Edge objects representing the smooth
+   *  interpolation of the boundary of the `triangulation`
```

Comment:
Would you be willing to add a sentence here or above stating what "smooth" means in this context?